### ISSUE TITLE
Question regarding DBGroup code and artifact release

### ISSUE THREAD

**Issue by NielsRogge** [liuxuexun/DBGroup#1]
Hi @liuxuexun 🤗

I'm Niels and work as part of the open-source team at Hugging Face. I discovered your work through Hugging Face's daily papers as yours got featured: https://huggingface.co/papers/2511.10003.
The paper page lets people discuss about your paper and lets them find artifacts about it (your models, datasets or demo for instance), you can also claim
the paper as yours which will show up on your public profile at HF, add Github and project page URLs.

I noticed in your abstract that you mentioned the code for DBGroup is available at https://github.com/liuxuexun/DBGroup. However, when trying to access the repository, the README content was not found.

Could you please confirm the status of the code release? If you are planning to release pre-trained model checkpoints for DBGroup or any associated datasets (such as the pseudo-labels you generate), it would be great to make them available on the 🤗 Hub to improve their discoverability and visibility. We can add tags so that people find them when filtering https://huggingface.co/models and https://huggingface.co/datasets.

If you're interested in hosting them, I'd be happy to provide guidance:
For models: https://huggingface.co/docs/hub/models-uploading
For datasets: https://huggingface.co/docs/datasets/loading

After uploading, we can also link the models and datasets to the paper page (read [here](https://huggingface.co/docs/hub/en/model-cards#linking-a-paper)) so people can discover your work.

You can also build a demo for your model on [Spaces](https://huggingface.co/spaces), we can provide you a ZeroGPU [grant](https://huggingface.co/docs/hub/en/spaces-gpus#community-gpu-grants), which gives you A100 GPUs for free.

Let me know if you're interested/need any help regarding this!

Kind regards,

Niels
ML Engineer @ HF 🤗
